CONTRIBUTING.md

```diff
@@ -27,7 +27,7 @@ Please include version numbers of all relevant libraries and Julia itself.
 - When possible write code to reuse memory. For example:
 ```julia
 function f(g, v)
-    storage = Vector{Int}(nv(g))
+    storage = Vector{Int}(uninitialized, nv(g))
```

Comment:
This can be updated later, but I think we want code that uses `zeros(T, nv(g))` over `uninitialized` because we don't want bugs that appear due to uninitialized memory. This hasn't been a problem, and shouldn't block this PR if you disagree, but can get handled at any time.

Reply:
We have `uninitialized` in several places, but they're all when we explicitly set every element of the vector. This provides a minor speedup.